src/gui/accountsettings.cpp

```diff
@@ -416,35 +416,35 @@ void AccountSettings::slotOpenMakeFolderDialog()
     }();
 
     if (!fileName.isEmpty()) {
-        const auto folderCreationDialog = new FolderCreationDialog(fileName, this); 
+        const auto folderCreationDialog = new FolderCreationDialog(fileName, this);
         folderCreationDialog->setAttribute(Qt::WA_DeleteOnClose);
         folderCreationDialog->open();
     }
 }
 
 void AccountSettings::slotEditCurrentLocalIgnoredFiles()
 {
-    QModelIndex selected = _ui->_folderList->selectionModel()->currentIndex();
+    const auto selected = _ui->_folderList->selectionModel()->currentIndex();
     if (!selected.isValid() || _model->classify(selected) != FolderStatusModel::SubFolder)
         return;
-    QString fileName = _model->data(selected, FolderStatusDelegate::FolderPathRole).toString();
+    const auto fileName = _model->data(selected, FolderStatusDelegate::FolderPathRole).toString();
     openIgnoredFilesDialog(fileName);
 }
 
 void AccountSettings::openIgnoredFilesDialog(const QString & absFolderPath)
 {
     Q_ASSERT(QFileInfo(absFolderPath).isAbsolute());
 
-    const QString ignoreFile = absFolderPath + ".sync-exclude.lst";
-    auto layout = new QVBoxLayout();
-    auto ignoreListWidget = new IgnoreListTableWidget(this);
+    const QString ignoreFile(absFolderPath + ".sync-exclude.lst");
```

Comment:
can you use `{ }` universal initialization syntax ?

Reply:
and yes, we should probably generalize and switch to this style to be more future proof and consistent

Reply:
I have used this syntax for empty QStrings and QStrings constructed this way